### PR TITLE
BUYT-16

### PR DESCRIPTION
**Pull request by maks2708**
Name: BUYT-16
Description: Footer made according to design. Hiding of the footer has been implemented.
Coverage: 72%
[Footer](https://buyticket.atlassian.net/browse/BUYT-16)

### CODE REVIEW

File: src/frontend/src/footer/style.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+}
+footer.open {
+  max-height: 500px; 
+}
+footer .footer-container{
+  display: flex;
+  justify-content: space-between;
+  max-width: 900px;
+  margin: 0 auto;
+}
```
Comment (VKuzmich) [inline src/frontend/src/footer/style.css:14]: please put styles which are in footer-container inside {}. 
also apply this code style to all places. 
example: 
if you have 
<div class="footer-container">
  <div class="logo"> 
  </div>
</div>

it should look like 
.footer-container{
  display: flex;
   
  .logo{
    justify-content: ...
  }
}

Comment (VKuzmich) [reply src/frontend/src/footer/style.css:14]: and try to do in all places where it is needed

Comment (VKuzmich) [reply src/frontend/src/footer/style.css:14]: in this case you won't overwrite other divs with the same names

Comment (VKuzmich) [reply src/frontend/src/footer/style.css:14]: and please, resolve conflicts

**Review by Stepan22-prog — APPROVED** [review]

**Pull request merged**